src/routes/resources/resources.handlers.ts

```
@@ -13,14 +13,43 @@ import type {
+      return c.json(JSON.parse(cacheData), HttpStatusCodes.OK);
+    }
   }
+
+  // const resources = await db.query.resources.findMany({
+  //   orderBy: (resources, { desc }) => desc(resources.createdAt),
+  //   where: (resources, { eq }) => eq(resources.isPublished, true),
+  // });
   const resources = await db.query.resources.findMany({
     orderBy: (resources, { desc }) => desc(resources.createdAt),
-    where: (resources, { eq }) => eq(resources.isPublished, true),
+    where: (resources, { and, eq, ilike }) =>
```

ESLint

[error] 'ilike' is defined but never used. Allowed unused args must match /^_/u.

(unused-imports/no-unused-vars)